include/openssl/trace.h

```diff
@@ -270,11 +270,19 @@ void OSSL_trace_end(int category, BIO *channel);
  *                42, "What do you get when you multiply six by nine?");
  */
 
-# define OSSL_TRACEV(category, args) \
+# if !defined OPENSSL_NO_TRACE && !defined FIPS_MODULE
+
+#  define OSSL_TRACEV(category, args) \
     OSSL_TRACE_BEGIN(category) \
         BIO_printf args; \
     OSSL_TRACE_END(category)
 
+# else
+
+#  define OSSL_TRACEV(category, args) ((void)0)
```

Comment:
Why the `((void)0)`?  Leaving the body empty should work.

Reply:
Not necessarily...  some compilers warn about extra `;`, which would be the consequence of an empty body

Reply:
Silly compilers ...